README.md

```diff
@@ -560,6 +560,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
 
 * [bake](https://github.com/bakeware/bake) - Configure, compile and share systems, toolchains and linux firmware.
```

Comment:
@GregMefford @fhunleth what if we just swap out backware for @entone's firmata lib like so?

```suggestion
* [firmata](https://github.com/entone/firmata) - Firmata protocol implemented in Elixir
```

Reply:
LGTM. 👍